kolibri/core/assets/src/api_resource.js

```diff
@@ -70,13 +70,15 @@ class Model {
 class Collection {
   /**
    * Create a Collection instance.
+   * @param {Object} params - Default parameters to use when fetching data from the server.
    * @param {Object[]|Model[]} data - Data to prepopulate the collection with,
    * useful if wanting to save multiple models.
    * @param {Resource} resource - object of the Resource class, specifies the urls and fetching
    * behaviour for the collection.
    */
-  constructor(data = [], resource) {
+  constructor(params = {}, data = [], resource) {
```

Comment:
Params are now passed preferentially into the constructor to allow for caching of Collections.